src/server/models/db/migrations/20221124224315_annual_review/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- AlterEnum
+-- This migration adds more than one value to an enum.
+-- With PostgreSQL versions 11 and earlier, this is not possible
+-- in a single migration. This can be worked around by creating
+-- multiple migrations, each migration adding only one value to
+-- the enum.
+
+
+ALTER TYPE "ListItemEvent" ADD VALUE 'ANNUAL_REVIEW_START_ONE_DAY_REMINDER_EMAIL_SENT';
+ALTER TYPE "ListItemEvent" ADD VALUE 'ANNUAL_REVIEW_START_ONE_WEEK_REMINDER_EMAIL_SENT';
+ALTER TYPE "ListItemEvent" ADD VALUE 'ANNUAL_REVIEW_START_ONE_MONTH_REMINDER_EMAIL_SENT';
+ALTER TYPE "ListItemEvent" ADD VALUE 'UNPUBLISH_ONE_DAY_POST_REMINDER_EMAIL_SENT';
+ALTER TYPE "ListItemEvent" ADD VALUE 'UNPUBLISH_WEEKLY_POST_REMINDER_EMAIL_SENT';
+ALTER TYPE "ListItemEvent" ADD VALUE 'UNPUBLISH_ONE_DAY_PROVIDER_REMINDER_EMAIL_SENT';
+ALTER TYPE "ListItemEvent" ADD VALUE 'UNPUBLISH_WEEKLY_PROVIDER_REMINDER_EMAIL_SENT';
+
+-- AlterTable
+ALTER TABLE "List" ADD COLUMN     "lastAnnualReviewStartDate" DATE,
+ADD COLUMN     "nextAnnualReviewStartDate" DATE NOT NULL DEFAULT NOW() + interval '1 year';
```

Comment:
is this just placeholder? I think this should be a scheduled task also, to determine the nextAnnualReviewStartDate. 

It should be a year-ish from the date the list first went "live" (i.e. when a citizen is first able to search for providers). This would be the date the first list item was published